src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -173,23 +173,13 @@
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
         (let [count-all (count submissions)
-              success-count (->> submissions (filter all-success?) count)
```

Comment:
I think the idea here is to find the submissions that have all of their workflows in a successful state - according to a definition of successful (see https://github.com/broadinstitute/firecloud-ui/blob/develop/src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs#L24). Since submission and workflow states can be different, we needed a way to bucket the submission based on *all* of the workflows inside it.